Langs/0405.lang

```diff
@@ -1,59 +1,59 @@
-RAM Installed=Instalovan� pam؜
+Disk Partition Type=Typ diskového oddílu
+RAM Installed=Instalovaná paměť
 Secure Boot=Secure Boot
-Storage Available=Dostupn� �lo�i�t�
+Storage Available=Dostupné úložiště
 TPM Version=Verze TPM
 Checking...=Kontroluji...
-64 Bit CPU=64bitov� CPU
-64 Bit OS=64bitov� OS
-32 Bit CPU=32bitov� CPU
-32 Bit OS=32bitov� OS
-Unable to Determine=Nelze ur�it
-Unable to Check List=Unable to Check List
-Error Accessing List=Error Accessing List
-Not Currently Listed as Compatible=Moment�ln� nen� v seznamu kompatibiln�ch
-Listed as Compatible=Je v seznamu kompatibiln�ch
-Cores=j�dra
-Threads=vl�kna
-GPT Detected=GPT detekov�na
-GPT Not Detected=GPT nebyla detekov�na
+64 Bit CPU=64bitový CPU
+64 Bit OS=64bitový OS
+32 Bit CPU=32bitový CPU
+32 Bit OS=32bitový OS
+Unable to Determine=Nelze určit
+Unable to Check List=Nelze ověřit seznam
+Error Accessing List=Seznam je nepřístupný
+Not Currently Listed as Compatible=Není v seznamu kompatibilních
+Listed as Compatible=Je v seznamu kompatibilních
+Cores=jádra
+Threads=vlákna
+GPT Detected=GPT detekována
+GPT Not Detected=GPT nebyla detekována
 Enabled=Povoleno
-Supported=Podporov�no
-Disabled / Not Detected=Vypnuto / Nepodporov�no
-Drive(s) Meet Requirements=disk(y) spl�uj�c� po�adavky
-TPM Missing / Disabled=TPM chyb� / vypnut
-Not Supported=Nepodporov�no
-Detected=detekov�no
+Supported=Podporováno
+Disabled / Not Detected=Vypnuto / Nepodporováno
+Drive(s) Meet Requirements=disk(y) splňující požadavky
+TPM Missing / Disabled=TPM chybí / vypnut
+Not Supported=Nepodporováno
+Detected=detekováno
 DirectX 12 and WDDM 3=DirectX 12 a WDDM 3
 DirectX 12 and WDDM 2=DirectX 12 a WDDM 2
 No DirectX 12, but WDDM2=WDDM2 bez DirectX 12
 DirectX 12, but no WDDM2=DirectX 12 bez WDDM2
 No DirectX 12 or WDDM2=Ani DirectX 12 ani WDDM2
-Test Build?=Testovac� verze?
-You're running a newer build than publically available!=M�te spu�t�nou nov�j� verzi ne� je ve�ejn� dostupn�!
-Up to Date=Aktu�ln�
-You're running the latest build!=M�te spu�t�nou posledn� verzi!
+Test Build?=Testovací verze?
+You're running a newer build than publically available!=Máte spuštěnou novější verzi než je veřejně dostupná!
+Up to Date=Aktuální
+You're running the latest build!=Máte spuštěnou poslední verzi!
 Unable to Check for Updates=Nelze zkontrolovat aktualizace
-Unable to load release data.=Unable to load release data.
-Invalid Data Received!=P�ijata chybn� data!
+Unable to load release data.=Nelze načíst data o verzi.
+Invalid Data Received!=Přijata chybná data!
 Invalid Release Tags Received!=Invalid Release Tags Received!
 Invalid Release Types Received!=Invalid Release Types Received!
-Update Available=Je dostupn� aktualizace
-An Update is Available, would you like to download it?=Je dostupn� aktualizace, chcete ji st�hnout?
\ No newline at end of file
+Update Available=Je dostupná aktualizace
+An Update is Availabe, would you like to download it?=Je dostupná aktualizace, chcete ji stáhnout?
```

Comment:
Please change Availabe to Available. Seems like you had older version of base file.